parm/archive/gefs_ocean.yaml.j2

```diff
@@ -0,0 +1,37 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".ocean.t" + cycle_HH + "z." %}
+
+gefs_ocean:
+    name: "GEFS_OCEAN"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_ocean.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set file_set = [] %}
+{% set membr = [] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do membr.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% for mem in membr %}
+    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                          '${RUN}':RUN,
+                          '${YMD}':cycle_YMD,
+                          '${HH}':cycle_HH,
+                          '${GRID}':"5p00",
+                          '${MEMDIR}':mem }) %}
+
+    {% set COMIN_OCEAN_NETCDF = COM_OCEAN_NETCDF_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+    # Select netcdf files to copy to the atardir
+    {% if RUN == "gefs" %}
+        {% if path_exists(COMIN_OCEAN_NETCDF) %}
+            {% for fhr in range(FHMIN_GFS + FHOUT_OCN_GFS, FHMAX_GFS, FHOUT_OCN_GFS) %}
+                {% set file_name = head ~ "5p00" ~ ".f" ~ '%03d'|format(fhr) ~ ".nc" %}
```

Comment:
The grid resolution should not be hard-coded here. The ocean product grid will vary depending on the ocean model resolution.